prep-css-flexbox/justify-content.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    div {
+      display: flex;
+      justify-content: center;
+    }
```

Comment:
Instead of setting all `div` elements to `display: flex` and `justify-content: center`, try only adding them to the `#card-deck`.